fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -647,10 +736,10 @@ private fun BottomDrawer(
             open = !drawerState.isClosed,
             onClose = onDismiss,
             fraction = {
-                if (drawerState.anchors.isEmpty()
-                ) {
+                if (drawerState.anchors.isEmpty() || drawerState.skipOpenState) {
                     0.toFloat()
-                } else {
+                }
+                else {
                     calculateFraction(
                         drawerState.anchors.entries.firstOrNull { it.value == DrawerValue.Closed }?.key!!,
                         drawerState.anchors.entries.firstOrNull { it.value == DrawerValue.Open }?.key!!,
```

Comment:
Get the second parameter as per the next state from close based on the parameter provided in DrawerState

Reply:
okayy